web/themes/custom/novel/novel.theme

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Novel Theme.
+ */
+
+/**
+ * Implements template_preprocess_page().
+ *
+ * @param mixed[] $variables
+ *   Theme variables.
+ */
+function novel_preprocess_page(array &$variables): void {
+  $data = [
+    'search-header-url' => '/search',
```

Comment:
Consider adding a TODO explaining that this url should be changed once we have a proper search result page.

Reply:
👍